container/sagemaker/serve.py

```diff
@@ -279,6 +300,12 @@ def _stop(self, *args):  # pylint: disable=W0613
         self._state = 'stopped'
         log.info('stopped')
 
+    def _wait_for_gunicorn(self):
+        while True:
```

Comment:
there should be a time limit on this -- maybe 30s?

Reply:
I used nginx to limit connection for /models